test/e2e/multi-cluster-deployment/README.md

```diff
@@ -37,7 +37,8 @@ $ kubectl config rename-context <default-wec1-context-name> kscore
 2. Run e2e test in your ocp cluster:
 
 ```
- export KUBESTELLAR_VERSION=0.21.2
+ export KUBESTELLAR_VERSION=0.22.0-rc2
```

Comment:
This needs to be mentioned in the release process doc. Not a change for this PR.

Reply:
I have included release process doc update in #2101, which also updates the self-references to RC3.